registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java

```diff
@@ -129,10 +140,6 @@
 	@Autowired
 	private RegistrationStatusService<String, InternalRegistrationStatusDto, RegistrationStatusDto> registrationStatusService;
 
-	/** The utilities. */
-	@Autowired
```

Comment:
Do we need to remove this ?

Reply:
These beans are unused in QualityCheckerStage